### PR TITLE
Property Docs - Merge Contents as Draft

### PR DESCRIPTION
**Pull request by SamBroner**
PR in the Fluid Framework main fork so that we get a preview of the docs

### CODE REVIEW

**Comment by tylerbutler** [comment]
@SamBroner I made a bunch of markdown lint fixes and just pushed directly to this branch. The staging site is now building as expected so you can preview the changes.

**Comment by tylerbutler** [comment]
I also reduced the file sizes of the PNGs using [PNGGauntlet](https://pnggauntlet.com/). Any chance we could avoid checking in binary files altogether?

File: docs/api-documenter.json [hunk 1]
```diff
@@ -2,5 +2,5 @@
   "outputTarget": "markdown",
   "newlineKind": "lf",
   "uriRoot": "/docs/apis",
-  "onlyPackagesStartingWith": "@fluidframework"
+  "onlyPackagesStartingWith": ["@fluidframework", "@fluid-experimental/property"]
```
Comment (tylerbutler) [inline docs/api-documenter.json:5]: You'll also need to edit docs/data/packages.yaml to include the relevant packages.

**Review by tylerbutler — APPROVED** [review]

**Comment by skylerjokiel** [comment]
Closing because it's stale

**Pull request closed without merging**